src/basic_bot/services/central_hub.py

```diff
@@ -99,19 +101,19 @@ async def send_state_update_to_subscribers(message_data):
         socket.close()
 
 
-async def notify_state(websocket="all", keysRequested=None):
+async def notify_state(websocket: Union[WebSocketServerProtocol, str] = "all", keysRequested: Optional[List[str]] = None) -> None:
```

Comment:
`Union[WebSocketServerProtocol, str]` why?   No string should be needed

Reply:
Ah!!  but central_hub did once have the dangerous feature that I long ago nixed to broadcast to all connected sockets.  Better to remove "all" here as a possibility.   If "all" was required, it would be better to have specified a `Literal` like this:  `Union[WebSocketServerProtocol, Literal["all"]]` instead of a `str`.